app/assets/stylesheets/application/donations/donations.scss

```diff
@@ -40,9 +40,34 @@
   width: 32%;
 }
 
+label[for="donation_amount_other"] {
+  margin-top: 1px;
+  vertical-align: middle;
+  width: 32%;
+  #donation_amount_other {
+    display: none;
+  }
+  input#other_text {
```

Comment:
Avoid using id selectors<br>Avoid qualifying id selectors with an element.<br>Selector `other_text` should be written in lowercase with hyphens